src/gui/editors/TimeLineWidget.cpp

```diff
@@ -59,35 +58,28 @@ TimeLineWidget::TimeLineWidget( const int xoff, const int yoff, const float ppb,
 	m_barLineColor( 192, 192, 192 ),
 	m_barNumberColor( m_barLineColor.darker( 120 ) ),
 	m_autoScroll( AutoScrollState::Enabled ),
-	m_loopPoints( LoopPointState::Disabled ),
-	m_behaviourAtStop( BehaviourAtStopState::BackToZero ),
 	m_changedPosition( true ),
 	m_xOffset( xoff ),
 	m_posMarkerX( 0 ),
 	m_ppb( ppb ),
 	m_snapSize( 1.0 ),
 	m_pos( pos ),
+	m_timeline{&timeline},
```

Comment:
Feels odd to use `{}` here.
```suggestion
	m_timeline(&timeline),
```

Reply:
I thought we preferred this style now, and it's [recommended by the Core Guidelines](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Res-list).

Reply:
> I thought we preferred this style now, and it's [recommended by the Core Guidelines](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Res-list).

I usually use `()` for the member initializer list and `{}` for initialization of local member variables. Not sure why I did it this way exclusively (probably because that's the only way I remember it being written :laughing:), but I'm fine if we use `{}` all round.